setup.py

```diff
@@ -39,6 +39,7 @@
       extras_require={
           'test': [
               'plone.app.testing',
+              'unittest2'
```

Comment:
@hvelarde because the tests uses that library but i don't have it without fixing setup.py.
For sure it's used in all the old tests, but i don't know if it can be replaced with standard unittest library.